<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainProgressFmtCfgDlg</class>
 <widget class="BaseDlg" name="MaintainProgressFmtCfgDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaintainProgressFmtCfgDlg</string>
  </property>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>266</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>266</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="PercentRBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>481</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>维护完成百分比</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="ItemCntRBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>481</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>维护完成项目数</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
